tests/ops/util/test_tcf_experience_contents.py

```diff
@@ -942,3 +942,39 @@ def test_add_same_data_use_to_different_systems(
             .id
             == 4
         )
+
+    @pytest.mark.usefixtures("ac_system_without_privacy_declaration")
+    def test_ac_systems_without_privacy_declarations(self, db):
```

Comment:
nice - this is good coverage.

a nit, but would it be worth covering the case where the AC vendor does have a "valid GVL" data use declaration (i.e. your `ac_system_with_privacy_declaration` fixture), even though we don't expect that to be common? just to make sure the expected overlay content is returned then. would also be a nice way to demonstrate/document what the expected overlay behavior is when that edge case (inevitably) rears its head in the wild 😄

Reply:
This is tested end-to-end in the `test_ac_system_not_in_tc_string`!   This test builds the TCF contents, and then builds the TC string from that.  There's an AC system and I am showing that while the vendor doesn't show up in the Vendor Consents section of the TC string, because it has valid GVL data uses, it's Purpose shows up in the purpose consents section.

Reply:
It was not clear though! I will add a separate section to this file where I'm asserting the breakdown of the TCF experience specifically.